SMS19/settings.py

```diff
@@ -58,13 +58,15 @@
 # EMAIL_BACKEND so allauth can proceed to send confirmation emails
 # ONLY for development/testing use console
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
-ACCOUNT_ADAPTER = "allauth.account.adapter.DefaultAccountAdapter"
+# EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'   This is used in production
+ACCOUNT_ADAPTER = "main.adapters.AccountAdapter"
```

Comment:
As Djano's user model does not have an attribute for name, so We are bound to make our own Custom adapter, which accepts the form and saves user and UserProfile object .